tutorial/part1-creating-a-custom-theme/create-a-theme-package.md

```diff
@@ -10,6 +10,10 @@ To create a package run the following command in the terminal:
 > npx frontity create-package my-first-theme
 ```
 
+{% hint style="info" %}
+The command above will not work with the latest npm version 7. Please make sure to use npm 6.
+{% endhint %}
+
```

Comment:
Thanks for pointing this out @kokiebisu, I'll test this to confirm that this is the case.

Reply:
This issue has been addressed: https://github.com/frontity/frontity/pull/702